src/plugins/data/public/ui/dataset_selector/_dataset_selector.scss

```diff
@@ -30,4 +30,20 @@
     flex: 1;
     align-self: center;
   }
+
+  &__button--truncate {
+    display: flex;
+    align-items: center;
+    max-width: 300px;
```

Comment:
Is this a magic number? Can't it use any OUI values?

Reply:
I just switched to a predefined EUI value. I also tried the the % but that will always leave a blank space in between the selector and the search bar.

Reply:
tried the `$euiSizeXXL`, however, it looks like it is too small
<img width="1212" alt="Screenshot 2024-09-21 at 12 33 12 AM" src="https://github.com/user-attachments/assets/850e51b9-4727-4c3d-83fc-d1b7903c59fb">

also tried `%` and it always has a gap
<img width="1213" alt="Screenshot 2024-09-21 at 12 33 53 AM" src="https://github.com/user-attachments/assets/91d41d17-1f63-4dad-9cd2-e430089503f6">